Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -628,6 +635,14 @@ extension PaymentSheet.IntentConfiguration.Mode {
     }
 }
 
+// MARK: - STPAnalyticsClientDelegate
+
+extension PlaygroundController: STPAnalyticsClientDelegate {
+    func analyticsClientDidLog(analyticsClient: StripeCore.STPAnalyticsClient, payload: [String: Any]) {
+        analyticsLog.append(payload)
```

Comment:
I think it might be legitimate failures, I ran your branch locally and noticed some Xcode warnings (purple) when I ran the failing tests but not on master (also didn't see failures on master).

You may need to wrap access to the `analyticsLog`  in a DispatchQueue.main.async. When I did this the test passed, the "Success!" text wasn't showing up in the failure case.

On a related note we should find a way to shorten the duration on these tests!

![CleanShot 2024-02-06 at 13 12 40@2x](https://github.com/stripe/stripe-ios/assets/88012362/62f408e0-5db7-42ad-b160-622ad5903786)

Reply:
I wonder if this is indicative of some other issue, e.g. it looks like getter on FlowController.paymentOption isn't being called on the main thread or something

Reply:
TY!